packages/testing/index.ts

```diff
@@ -26,5 +26,6 @@ export {
   assertSucceeds,
   initializeAdminApp,
   initializeTestApp,
+  initializeFirestoreTestApp,
```

Comment:
You need to remove this to get the CI to pass.

Reply:
done